_posts/2019-12-20-this-blog-post-contains-important-information-for-some-percentage-of-its-readers.markdown

```diff
@@ -0,0 +1,150 @@
+---
+layout: epic
+title: This Blog Post Contains Important Information For Some Percentage of its Readers
+date: 2019-12-20
+author: myk
+categories: [mental-health, culture, diversity, autism, neurodiversity]
+comment_id: 598
+---
+
+You are a software engineer.
+
+You consider yourself an introvert, and you subscribe to the idea of "engineering time" where you like to work for
+extended uninterrupted periods because interruptions wreck you. You are used to being misunderstood. Ever since you
+can remember the people around you have been kind of baffling: they constantly fail to notice stuff that's really
+obvious and important to you, and then they have the audacity to get frustrated with you for not understanding
+them.
+
+But whatever, you can deal with this, right? This is just how life goes, right? Everyone's like this, right?
+
+Right?
+
+<!-- more -->
+
+It's really been like this as long as you can remember. Maybe even as early as kindergarten, when you distinctly
+remember feeling "Wow, I guess all these kids must already know each other?" because how else would they know how
+to just start playing together? You might have gotten really, really good at languages -- or you might have
+eschewed the humanities entirely and focused on things like Math and Science. Regardless, you were in it for the
+_systems_ -- grammar just _makes sense_ to you, in the same way software engineering eventually would. Math, you
+finally understood, was just a specialized language for referring to the relations between things -- sure, okay!
+
+Socially things were always a little weird for you - people were never _quite_ behaving in ways that made sense,
+and especially when younger you probably socialized by getting 'adopted' by some other kid who seemed to know what
```

Comment:
If there's further reading for "adopted" here, that'd be great to link to 👍

Reply:
I'm going to hold off on that since any link in this area would potentially spoil the surprise of the punch line below, right? If I were to start linking out to every example this whole thing would be links, and I def want to provide those resources but I think this is not the place to do that, at least for my current strategy.